stat/stat_test.go

```diff
@@ -1434,10 +1434,14 @@ func TestCDF(t *testing.T) {
 }
 
 func TestQuantile(t *testing.T) {
+	const tol = 1e-12
+
 	cumulantKinds := []CumulantKind{
 		Empirical,
 		LinInterp,
+		Linear,
 	}
+
```

Comment:
```suggestion
```

Reply:
Does this suggested amendment improve either the readability or compatibility of the code?